projects/angular/src/data/datagrid/datagrid-row.ts

```diff
@@ -189,6 +189,15 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
     return this._detailCloseLabel ? this._detailCloseLabel : this.commonStrings.keys.close;
   }
 
+  private _rowAriaLabel = '';
```

Comment:
This gives applications the ability to create meaningful strings to be used in the aria-label on the selection inputs (checkbox/radio) for both single and multi select modes. The fallback is a common string.